source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -1454,6 +1455,18 @@ message TransformationReplaceLoadStoreWithCopyMemory {
   InstructionDescriptor store_instruction_descriptor = 2;
 }
 
+message TransformationReplaceOpSelectWithConditionalBranch {
+
+  // A transformation that takes an OpSelect instruction and replaces
+  // it with a conditional branch and an OpPhi instruction.
+
+  // The result id of the OpSelect instruction
+  uint32 select_id = 1;
+
+  // Fresh ids for the new blocks
+  UInt32Pair new_block_ids = 2;
```

Comment:
Not a strict requirement but it would probably be better to use two separate fields. Something like `true_block_fresh_id` and `merge_block_fresh_id`.

Reply:
Yes, I agree